packages/node-cli/src/services/FileSystemService.js

```diff
@@ -20,7 +20,20 @@ module.exports = class FileSystemService {
 
 		const availableDirectories = getDirectories(parentFolder);
 
-		return [parentFolder, ...availableDirectories];
+		return availableDirectories;
```

Comment:
Actually now this function does exactly what it says, return the directories inside of the parentFolder, but it is used also in the ImportObjectsCommandGenerator.js , and there is expecting the parantFolder in the returned array. One possible fix can be adding there (ImportObjectsCommandGenerator.js) the '/Objects' folder as one of options to pick.

Reply:
Agree. Adding the 'Objects' folder in ImportObjectsCommandGenerator.js